alfresco-java-rest-api/alfresco-java-rest-api-lib/alfresco-event-gateway-api/src/main/java/io/swagger/configuration/ApiKeyRequestInterceptor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021-2021 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I am not sure about the license header you are using. Are we sure these enterprise files are licensed under Apache License?

Reply:
It's part of the generation script, running mvn license:format, it's picking up the set one for the other generated clients, like: https://github.com/Alfresco/alfresco-java-sdk/blob/develop/alfresco-java-rest-api/alfresco-java-rest-api-lib/generated/alfresco-core-rest-api/src/main/java/org/alfresco/core/handler/ActionsApi.java#L1-L20